applications/pelion_client/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Please verify the license headers in this PR, a lot of files still use the old license ref, `LicenseRef-Nordic-5-Clause` should be used instead.

Reply:
fixed